fern/pages/fern-docs/content/customize-api-ref.mdx

```diff
@@ -0,0 +1,190 @@
+---
+title: Customize API Reference layout
+description: Customize your API Reference's naming, ordering, and structure.  
+---
+
+When you include an API (either the Fern or an OpenAPI definition) in your `docs.yml` file, you can customize how the endpoints and sub-packages are displayed in the sidebar navigation. By default, the reference will generate a navigation hierarchy based on the structure of the API spec, but several customizations can be configured.
+
+## Ordering the API Reference
+
+### Flattening sections
```

Comment:
Flattening sections isn't about "Ordering the API Reference" is it? This is just how I get rid of the API Reference header?

Reply:
Agree!